nixos/modules/services/networking/anubis.nix

```diff
@@ -245,6 +269,25 @@ in
   };
 
   config = lib.mkIf (enabledInstances != { }) {
+    warnings = [
+      "RuntimeDirectory is going to be migrated from `anubis` to `anubis/anubis-<instance name>`, update BIND and METRICS_BIND to ${runtimeDirectoryPrefix "<instance name>"} if using unix sockets"
+    ];
```

Comment:
Sorry, I've just realised I've made a review mistake here.  We shouldn't produce warnings that users can't do anything about, because this can lead to [alarm fatigue](https://en.wikipedia.org/wiki/Alarm_fatigue) where users get used to ignoring warnings.  Could you make it so that the warning is only shown if the update hasn't been done?

Reply:
Right, I'll look into it.

Reply:
See https://github.com/NixOS/nixpkgs/pull/457355.